src/actions/newsActions.js

```diff
@@ -76,10 +79,36 @@ export const searchNews = (search) => async (dispatch) => {
   }
 };
 
+export const create =
+  ({ title, userId, content, category, readTime, coverImage, communityId }) =>
+  async (dispatch) => {
+    try {
+      const { richtext } = await api.richText.create();
+
+      const response = await api.news.create({
+        title,
+        readTime,
+        communityId,
+        creator: userId,
+        news: coverImage,
+        category: [category],
+        richtextId: richtext.id,
+      });
+
+      const { id: newsId } = response?.data?.data || {};
+      await createRichText(content, richtext.id, dispatch);
+
+      return Promise.resolve({ newsId });
+    } catch (error) {
+      return Promise.reject(getErrorMessage(error));
+    }
+  };
+
 export const createNews =
```

Comment:
Looks like this function is not used anymore. So may be better to clean it up

Reply:
leave it for now, i will clean up this file in next pr's